src/ChowDSP.hpp

```diff
@@ -251,7 +251,7 @@ class AAFilter {
 	 * @param osRatio: The oversampling ratio at which the filter is being used
 	 */
 	void reset(float sampleRate, int osRatio) {
-		float fc = 0.98f * (sampleRate / 2.0f);

```

Comment:
Even with quite a steep filter, 0.98 of Nyquist is too close to really be effective in reducing aliasing. The value 0.85 was found empirically to provide a good balance of reducing aliasing without effecting higher harmonics too strongly.